src/Form/_FormCheckbox.scss

```diff
@@ -0,0 +1,157 @@
+.pgn__form-checkbox {
+  font-size: $font-size-base;
+  line-height: $line-height-base;
+  display: block;
+  flex-shrink: 0;
```

Comment:
I thought flex-shrink was 0 by default?

Reply:
Defaults to 1: https://developer.mozilla.org/en-US/docs/Web/CSS/flex-shrink#values